tools/runtime_CLI.py

```diff
@@ -2602,6 +2605,11 @@ def do_set_crc32_parameters(self, line):
     def complete_set_crc32_parameters(self, text, line, start_index, end_index):
         return self._complete_crc(text, 32)
 
+    def do_set_rss_key(self, line):
+        "Change the RSS key for the Toeplitz hash: set_rss_key <name> <rss key in hex string>"
+        name, rss_key = line.split()
+        rss_key = bytearray.fromhex(rss_key)
```

Comment:
I'd suggest doing some extra client-side validation here on the key (e.g., make sure the length is a multiple of 4). That can help provide more helpful error messages.

Reply:
Done.